### DISCUSSION TITLE
How to custom prefix

### DISCUSSION THREAD

**Discussion by lilHinhbongems** [jagrosh/MusicBot#1420] [Questions]
How can I custom the prefix ? neither "!!' nor "@mention" .

**Comment by MichailiK** [comment]
The [config.txt](https://jmusicbot.com/config/) has a line where you can specify the prefix.

![image](https://github.com/jagrosh/MusicBot/assets/39029839/e6707c9c-2488-4d4a-850b-1a148f84f07c)
